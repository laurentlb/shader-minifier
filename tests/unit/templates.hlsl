template <typename T>
struct literal {
  T value;
};

template <typename T>
literal<T> DoSomething(T arg, literal<T> b) {
  T a = arg;
  literal<T> c = b;
  return c;
}
