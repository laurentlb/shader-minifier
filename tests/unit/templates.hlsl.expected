template<typename T>
struct literal{
  T value;
};
template<typename T>
literal<T> DoSomething(T arg,literal<T> b)
{
  return b;
}
